desktop/ui/src/main/java/org/datacleaner/widgets/visualization/JobGraphKeyListener.java

```diff
@@ -49,25 +50,27 @@ public JobGraphKeyListener(JobGraphContext graphContext) {
     public void keyTyped(KeyEvent e) {
         // react to DEL key strokes and delete components that are selected.
         if (e.getKeyChar() == KeyEvent.VK_DELETE) {
-            final Object vertex = _graphContext.getSelectedVertex();
-            logger.debug("Registered typed DEL. Vertex: {}", vertex);
-            if (vertex != null) {
+            final Set<Object> vertices = _graphContext.getSelectedVertices();
+            logger.debug("Registered typed DEL. Vertices: {}", vertices);
+            if ((vertices != null) && (!vertices.isEmpty())) {
```

Comment:
I also thought so, but see message of commit no. 2:  Checking if selected vertices set is empty (not only null) :).